pkg/agent/flowexporter/connections/conntrack_windows.go

```diff
@@ -0,0 +1,41 @@
+// +build windows
+
+package connections
+
+import (
+	"net"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+)
+
+var _ ConnTrackPoller = new(connTrackPoller)
```

Comment:
@tnqn do we have coding guidelines about this? I believe this is sometimes used as a way to explicitly state that  `connTrackPoller` implements the `ConnTrackPoller` interface. I kind of feel like it is somewhat of a Go anti-pattern and I'd rather have a comment on the `connTrackPoller` type that states that it implements `ConnTrackPoller`, but I am not a Go expert by a long shot...

Reply:
I just want to add that my intention was to use it as a sanity check to make sure that the struct implements interface completely and its pointer could be as interface object.

Reply:
It's a Go recommended way to add a static check when there are no static conversions already present in the code.
https://golang.org/doc/effective_go.html#blank_implements
It's also widely used in K8s: https://github.com/kubernetes/kubernetes/blob/15e95e48964d4e800f7210a249f9122ee7d79fcc/pkg/registry/core/service/allocator/storage/storage.go#L59

Reply:
I never noticed that we do the same thing in other places in Antrea :)

In this case I suggest that we do the same thing as in that k8s code snippet you shared, and add a comment before the declaration, e.g:
> connTrackPoller implements the ConnTrackPoller interface

to make it more obvious what the declaration is for...